spec/support/webmock_ruby24_bridge.rb

```diff
@@ -0,0 +1,10 @@
+# Ruby 2.4.0 removed the closed? check in the conditional in: s.close if !s.closed?
+# Webmock was changed to add close to StubSocket along with another change..
+# https://github.com/ruby/ruby/commit/f845a9ef76c0195254ded79c85c24332534f4057
+# https://github.com/bblimke/webmock/commit/8f2176a1fa75374df55b87d782e08ded673a75b4
+# We should upgrade webmock but that requires some re-recording of cassettes (I think)
+# and maybe other things.
+class StubSocket
+  def close
+  end
+end
```

Comment:
note, @cben I'd love your feedback on the failures in https://github.com/ManageIQ/manageiq/pull/14270 since it looks like much of the failures are VCR cassettes you may know how to re-record.